airgun/entities/domain.py

```diff
@@ -0,0 +1,192 @@
+from navmazing import NavigateToSibling
+from wait_for import wait_for
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.domain import (
+    DomainCreateView,
+    DomainListView,
+    DomainEditView
+)
+
+
+class DomainEntity(BaseEntity):
+    _DEFAULT_ORG = 'Default Organization'
```

Comment:
These constants already defined in robottelo and basically all such data should be checked on a test level. Framework should only pass data and return it back to the tests